### DISCUSSION TITLE
How to use osgEarthProcedural for splatting?

### DISCUSSION THREAD

**Discussion by rmk177** [pelicanmapping/osgearth#1918] [Support Q&A]
Good day master version doen't work with splatting. Is there any good .earth file how to make it?
I have error 
glLinkProgram 0x7f1c64958e50"SplatLayer" FAILED
Program "SplatLayer" infolog:
Fragment info
-------------
0(85) : error C3002: call to undefined function "vec4 oe_terrain_getNormalAndCurvature(vec2);"

**Comment by gwaldron** [comment]
Don't use it yet -- still under development until further notice.

**Comment by rmk177** [comment]
Ok, I''ll wait, but please fix texture coordinates when osgEarthSplat works.
Also I start to learn terrior branch and find out that you use biomes shapes to place trees. How cool!
